src/Symfony/Component/Config/Definition/ArrayShapeGenerator.php

```diff
@@ -100,6 +100,21 @@ private static function handleNumericNode(NumericNode $node): string
         return 'int|float';
     }
 
+    private static function handleEnumNode(EnumNode $node): string
+    {
+        $enumFqcn = $node->getEnumFqcn();
+
+        if ($enumFqcn && is_subclass_of($enumFqcn, \BackedEnum::class)) {
+            $values = array_column($enumFqcn::cases(), 'value');
+
+            return implode('|', array_map(static function ($value) {
+                return \is_string($value) ? '"'.$value.'"' : $value;
```

Comment:
```suggestion
                return \is_string($value) ? var_export($value, true) : $value;
```

`"test` will become `""test"`, `"` must be escape, so i think `var_export` is good solution for it, but it using `'` to wrap string, or maybe `addslashes($value, '"')` but i dont really like it

Ref: https://phpstan.org/r/924d234a-ce8c-46e4-9374-ff940704394d

Reply:
updated